pkgs/jni/test/jlist_test.dart

```diff
@@ -91,7 +91,7 @@ void run({required TestRunnerCallback testRunner}) {
   testRunner('contains', () {
     using((arena) {
       final list = testDataList(arena);
-      // ignore: iterable_contains_unrelated_type
+      // ignore: collection_methods_unrelated_type
```

Comment:
Why is this triggering?

Reply:
because `"1" != "1".toJString()` but here I'm testing the fact that calling `contains` with non-`JObject`s returns `false`.